src/main/java/io/rainfall/configuration/ConcurrencyConfig.java

```diff
@@ -57,11 +60,19 @@ public int getThreadCount() {
     return threadCount;
   }
 
+  public ScheduledExecutorService getScheduledExecutorService() {
```

Comment:
I don't like prefixing methods that create heavyweight objects with ```get``` as those methods generally are presumed to be lightweight (rarely more than a simple return) and the returned objects don't need to be lifecycled.

I personally prefer the ```create``` prefix in this case.